rasa_core/channels/facebook.py

```diff
@@ -88,12 +88,14 @@ def hello():
                 if request.args.get("hub.verify_token") == self.fb_verify:
                     return request.args.get("hub.challenge")
                 else:
+                    logger.debug("Failure, invalid token")
```

Comment:
let's make them `logger.warn` so they always show up and be a little more verbose (keep in mind that not everyone knows that code and what happened when that message gets raised - so the message should explain what is going wrong.)

Reply:
good point 👍